pkg/state/state_shared_dp.go

```diff
@@ -53,13 +54,20 @@ type stateSharedDp struct {
 	stateSkel
 }
 
+type sharedDpRuntimeSpec struct {
+	runtimeSpec
+	CPUArch string
```

Comment:
Do you require `CPUArch` and `OSVer` ?
I noticed this PR uses only OSName in the manifest templates.

Reply:
I'm not using them but I thought it made sense to keep them as they're all populated by the same `GetNodeAttributes()` function.

Reply:
IMO, it's better to keep only required code. We can extend sharedDpRuntimeSpec  in the future